akka-projection-core/src/main/scala/akka/projection/Projection.scala

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
+
+import akka.Done
+import akka.actor.ClassicActorSystemProvider
+import akka.annotation.ApiMayChange
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+@ApiMayChange
+trait Projection {
+  def start()(implicit systemProvider: ClassicActorSystemProvider): Unit
+  def stop(): Future[Done]
```

Comment:
I was considering adding a Java API, but I'm holding it for the moment. Let's see how this evolve. 

I won't expect users to implement this trait, although it should be open for extension. We can later decide if we need a Java variant. Maybe an abstract class that implements this one. To be decided.

Reply:
that's fine